src/components/ProfilePage.tsx

```diff
@@ -153,6 +161,33 @@ export const ProfilePage = (props: ProfilePageProps) => {
         </>
     ) 
 
+    const onCropComplete = (croppedArea: any, croppedAreaPixels: any) => {
+		setCroppedArea(croppedAreaPixels);
+	};
```

Comment:
Unused first argument. Also, please specify the type for `croppedAreaPixels`

Reply:
The first unused argument was required from the library even when not being used.